content/silver/concepts/automatic-attributes.md

```diff
@@ -442,6 +443,21 @@ top::Type ::= inputType::Type outputType::Type
 
 As with equality attributes, `==` is used to compare any children that don't have the propagated partial attribute.
```

Comment:
Should this be " ... `==` can be used to" instead of "is" ?

Reply:
No, I am discussing what the generated implementation does here.